portfolio/src/main/webapp/script.js

```diff
@@ -32,12 +32,22 @@ function addRandomImage() {
   imageContainer.appendChild(image_element);
 }
 
-function loadTasks() {
+function commentsSection(){
+    var amount = document.getElementById("max-comments").value;
+    loadTasks(amount);
+}
+
+async function loadTasks(amount) {
+
   fetch('/data').then(response => response.json()).then((tasks) => {
-    const taskListElement = document.getElementById('task-list');
-    tasks.forEach((task) => {
-      taskListElement.appendChild(createTaskElement(task));
-    })
+    taskListElement = document.getElementById('task-list');
+    taskListElement.innerHTML = '';
+    for (let x = 0; x<amount; x++) {
+        taskListElement.appendChild(createTaskElement(tasks[x]));
+    }
+    // tasks.forEach((task) => {
+    //   taskListElement.appendChild(createTaskElement(task));
+    // })
```

Comment:
Why is this commented out? If you no longer need this code delete it, otherwise add a comment saying why it's here (ex; "need to fix X and Y")